gradio/cli/commands/reload.py

```diff
@@ -126,7 +126,11 @@ def main(
             GRADIO_WATCH_DEMO_PATH=str(path),
         ),
     )
-    popen.wait()
+    if popen.poll() is None:
+        try:
+            popen.wait()
+        except (KeyboardInterrupt, OSError):
+            print("gradio-cli: Waiting gradio main thread cleaning.")
```

Comment:
Hi @Tiger3018 ! This contribution looks good to me but I don't think we need print statement here

Reply:
In the original implementation, the typer module will print `Aborted!` (See [this line](https://github.com/tiangolo/typer/blob/f6ba077dca84dfb567a8ee7e4580b0657d80878a/typer/core.py#L204) in typer module) and silently exit. This PR will remove the `Aborted!` echo and I don't know if an alternative text should be there.

Maybe we should just follow the implementation of gradio module, and use `pass` in this line instead?

Reply:
Yes, `pass` is what I had in mind

Reply:
Pushed that up so we can merge this in. Thanks!